api/block/schema.py

```diff
@@ -0,0 +1,25 @@
+import graphene
+from graphene import relay,Schema
+from graphene_sqlalchemy import (
+    SQLAlchemyObjectType, 
+    SQLAlchemyConnectionField
+)
+from api.block.models import Block as BlockModel
+
+class Block(SQLAlchemyObjectType):
+    class Meta:
+        model = BlockModel
+        interfaces = (relay.Node,)
```

Comment:
Avoid using relay Nodes as they add extra fields edges and Node to the query payload and response

Reply:
I totally agree. I have implemented the requested changes.